.travis.yml

```diff
@@ -29,6 +29,8 @@ gemfile:
 
 services:
   - mongodb
+  - mysql
+  - postgresql
```

Comment:
nop, in fact on travis  `precise` **env** (default) those services are loaded by default, but with `trusty` env (to have phantom2) we must list all service that have to be started